e2e/process-services-cloud/people-group-cloud-component.e2e.ts

```diff
@@ -66,8 +67,10 @@ xdescribe('People Groups Cloud Component', () => {
             clientActivitiAdminRoleId = await rolesService.getClientRoleIdByRoleName(groupActiviti.id, clientId, CONSTANTS.ROLES.ACTIVITI_ADMIN);
             clientActivitiUserRoleId = await rolesService.getClientRoleIdByRoleName(groupActiviti.id, clientId, CONSTANTS.ROLES.ACTIVITI_USER);
 
-            apsUser = await identityService.createIdentityUser();
+            testUser = await identityService.createIdentityUser();
             apsUserRoleId = await rolesService.getRoleIdByRoleName(CONSTANTS.ROLES.APS_USER);
+            await identityService.assignRole(testUser.idIdentityService, apsUserRoleId, CONSTANTS.ROLES.APS_USER);
+            apsUser = await identityService.createIdentityUser();
```

Comment:
What is the difference between those users? TestUser apsUser activitiUser?

Reply:
testUser , we are creating to do the tests. apsUser and activitiUser are assigned specific roles to test the role filtering in the PeopleGroupCloudComponent.